helm_deploy/hmpps-electronic-monitoring-datastore-api/values.yaml

```diff
@@ -11,8 +11,35 @@ generic-service:
 
   ingress:
     enabled: true
-    host: app-hostname.local # override per environment
+    annotations:
+      nginx.ingress.kubernetes.io/custom-http-errors: "418"
+    hosts:
+      - app-hostname.local # override per environment
+    path: /
+    healthPath: /health
     tlsSecretName: hmpps-electronic-monitoring-datastore-cert
+    modsecurity_enabled: true
```

Comment:
if you just want to enable mod security and default all the values from the generic service helm chart you'd be better to just set
```
    modsecurity_enabled: true
    modsecurity_github_team: "hmpps-electronic-monitoring"
```
and not override anything else.  You don't need to copy everything from [the generic service values file](https://github.com/ministryofjustice/hmpps-helm-charts/blob/ee741f5c600aea78eb345c94e71c0bc394652af7/charts/generic-service/values.yaml#L68)